bin/test-entry-creation.js

```diff
@@ -0,0 +1,36 @@
+require('dotenv').config();
+
+const { createNewEntry, publishEntry } = require('../lib/content');
+const { waitUntilContentIsDelivered } = require('../lib/delivery');
+
+const cmaToken = process.env.CMA_TOKEN;
+const spaceId = process.env.SPACE_ID;
+const cdaToken = process.env.CDA_TOKEN;
+
+run();
+
+async function run () {
+  const publishStartedAt = process.hrtime();
+
+  // Create a new entry
+  // Publish it
+  // Send requests to CDA until published version is delivered
+  // Print out the time that took
+  const [entryId, entryVersion] = await createNewEntry({ cmaToken, spaceId });
+
+  await publishEntry({ cmaToken, spaceId, entryId, version: entryVersion });
+  await waitUntilContentIsDelivered({
+    entryId,
+    expectedRevision: 1,
+    spaceId,
+    cdaToken
+  });
+
+  console.log(msSince(publishStartedAt));
+}
+
+function msSince (start) {
```

Comment:
We use this function in more than one test and probably in every other test. Can we move it back to a utility module?

Reply:
:+1: moved